db/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "jmsormond@gmail.com": {
+        "bestTime": "m",
```

Comment:
Is there a set of definitions to what `m` actually *means* in terms of `bestTime`? Perhaps you can create a module do all these definitions so when we flesh out the algorithm the definitions are easily importable and flexibly interacted with.

Additionally, if you have a compartmentalized module, you can more easily test it. Which I think is important to have here given that we're dealing with user settings.